### DISCUSSION TITLE
Checkpoint cannot be selected, keep loading indefinitely

### DISCUSSION THREAD

**Discussion by StableInfo** [vladmandic/sdnext#1420] [Q&A]
![image](https://github.com/vladmandic/automatic/assets/136593585/c8158dac-f1e3-4e61-a778-ddb951cd1ef1)

It could because I deleted some models and or moved them inside the  auto local repo,
Anyway it keeps loading, 
It can generate images, Except I cant select models now (exepct if I cheat with XYZ tool)
Any idea what I can do?

**Comment by vladmandic** [comment]
#1113

**Comment by ghost** [comment]
I only get this bug when I touch or somehow edit the ui before the initial load is done. If I let it load and then I edit the settings everything is fine but earlier and it will freeze
